auto_editor/Fix1003Rule.py

```diff
@@ -17,5 +17,103 @@ def change_type(self) -> ChangeTypeEnum:
     def run_rule(self, project: StructuredProjectSource,
                  warning_first_line: int, warning_last_line: int, file_path: str) -> StructuredProjectSource:
         # Todo: add rule here
-        print("Add rule here")
+        first_time = True
+        warning_code, prefix = "", ""
+        tmp_dict = project.paths_to_lines
+        all_lines = tmp_dict[file_path]
+
+        for i in range(warning_last_line+1, len(all_lines)):
+            print("warning code:",all_lines[i].code)
+            if ";" not in all_lines[i].code:
+                now_code = self.strip_determin(first_time, all_lines[i])
+                warning_code = warning_code + now_code
+                warning_code = warning_code.replace("\n", "")
+                first_time = False
+            else:
+                now_code = self.strip_determin( first_time, all_lines[i])
+                warning_code = warning_code + now_code + "\n"
+                prefix, new_code = self.remove_function_info(warning_code)
+                new_code = prefix + new_code
+                print("new_code:",new_code)
+
+                self.replace_useless_multiple_line(warning_last_line, i, all_lines)
+                all_lines[warning_last_line+1].code = new_code
+                self.test_print( all_lines, warning_last_line)
+                tmp_dict[file_path] = all_lines
+                break
+
+        project.paths_to_lines[file_path] = tmp_dict[file_path]
         return project
+
+    def strip_determin(self,first_time,lines):
+        if first_time == False:
+            now_code = lines.code.strip()
+        else:
+            now_code = lines.code
+
+        return now_code
+
+    def replace_useless_multiple_line(self,warning_last_line,i,all_lines):
+        for j in range(warning_last_line + 1, i + 1):
+            temp = all_lines[j]
+            temp.code = ""
+            all_lines[j] = temp
```

Comment:
Instead of editing existing lines, please always add/remove line items, so that a new ID is created/old ID is removed. Otherwise the change won't be noticed by the change tracker